actors/albert/main.js

```diff
@@ -101,7 +105,9 @@ function getCategoryProductsUrl(category, pageNumber = 0, pageSize = 50) {
       pageNumber,
       pageSize,
       filterFlag: true,
-      plainChildCategories: true
+      plainChildCategories: true,
+      includeSponsoredProducts: false,
+      keyword: `chips` // not sure what it means, it's hardcoded in the client code
```

Comment:
Are those really needed? IMHO changing just the saved query ID should be enough.